agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -117,6 +118,35 @@ async def add_variant_based_on_image(
     )
     await engine.save(db_app_variant)
 
+    # Create testset for demonstration purposes
+    if os.environ["FEATURE_FLAG"] == "demo":
+        csvdata = [
+            {
+                "country": "Nigeria",
+                "correct_answer": "The capital of Nigeria is Abuja.",
+            },
+            {
+                "country": "France",
+                "correct_answer": "The capital of France is Paris.",
+            },
+            {
+                "country": "Germany",
+                "correct_answer": "The capital of Nigeria is Berlin.",
+            },
+            {
+                "country": "Italy",
+                "correct_answer": "The capital of Italy is Rome.",
+            },
+        ]
```

Comment:
Let's add unknown countries :p 

Nauru Tuvalu
Capital: Funafuti

Brunei
Capital: Bandar Seri Begawan

Kiribati
Capital: Tarawa

Comoros
Capital: Moroni

Kyrgyzstan
Capital: Bishkek

Azerbaijan
Capital: Baku

Reply:
Roger that, captain. 🫡